addon/webextension/content-script.js

```diff
@@ -36,14 +36,14 @@ function findAndReplace(wordList) {
     });
   }
 
-    // append the ones we saw as a result
-    if (document.querySelector('#wanted')) {
-      document.querySelector('#wanted').innerText="wanted: " + Array.from(wordList);
-    }
-    if (document.querySelector('#wanted')) {
-      document.querySelector('#seen').innerText="seen: " + Array.from(seen);
-    }
+  // append the ones we saw as a result
+  if (document.querySelector("#wanted")) {
+    document.querySelector("#wanted").innerText = "wanted: " + Array.from(wordList);
   }
+  if (document.querySelector("#wanted")) {
```

Comment:
@gregglind Is this kosher? Or should this be `if (document.querySelector("#seen")) { ... }`?